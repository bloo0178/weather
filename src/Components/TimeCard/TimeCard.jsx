import React from "react";

export function TimeCard(props) {
  const { temp, iconSrc, time, tempUnit } = props;
  return (
    <div>
      <div>{temp} {tempUnit}</div>
      <div>
        <img src={iconSrc} alt="Current weather icon" />
      </div>
      <div>{time}</div>
    </div>
  );
}
