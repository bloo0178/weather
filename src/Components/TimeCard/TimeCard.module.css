.Container {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    background-color: black;
    opacity: 1;
    color: white;
    border: 3px solid black;
    border-radius: 10px;
    padding: 1rem;
    padding-left: 2rem;
    padding-right: 2rem;
}