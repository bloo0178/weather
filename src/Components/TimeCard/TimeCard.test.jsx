import React from "react";
import {TimeCard} from "./TimeCard";
import { shallow } from "enzyme";

describe("<TimeCard />", () => {
	test("renders correctly", () => {
		let wrapper = shallow(
			<TimeCard
				iconSrc="../../common/icons/10n.svg"
				time="Currently"
				temp="85"
			/>
		);
		expect(wrapper).toMatchSnapshot();
	});
});
