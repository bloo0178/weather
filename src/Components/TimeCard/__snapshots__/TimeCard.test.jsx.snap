// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeCard /> renders correctly 1`] = `
<div
  className="Container"
>
  <div>
    85
  </div>
  <div>
    <img
      alt="Current weather icon"
      src="../../common/icons/10n.svg"
    />
  </div>
  <div>
    Currently
  </div>
</div>
`;
